kura/pom.xml

```diff
@@ -69,7 +69,6 @@
         <module>org.eclipse.kura.wire.provider</module>
         <module>features</module>
         <module>emulator</module>
-        <module>test</module>
```

Comment:
My original idea was to add that step in the hudson build and in the build-all script. But we can do this way as well